src/datasheet.js

```diff
@@ -0,0 +1,46 @@
+const data = {
+  '': 0,
+  ' ': 0,
+  '\n': 0,
+  '\t': 0,
+  0: 0,
+  1: 0,
+  2: 0,
+  3: 0,
+  4: 0,
+  5: 0,
+  6: 0,
+  7: 0,
+  8: 0,
+  9: 0,
+  a: 1,
+  b: 3,
+  c: 3,
+  d: 2,
+  e: 1,
+  f: 4,
+  g: 2,
+  h: 4,
+  i: 1,
+  j: 8,
+  k: 5,
+  l: 1,
+  m: 3,
+  n: 1,
+  o: 1,
+  p: 3,
+  q: 10,
+  r: 1,
+  s: 1,
+  t: 1,
+  u: 1,
+  v: 4,
+  w: 4,
+  x: 8,
+  y: 4,
+  z: 10
+}
+
+module.exports = {
+  datasheet: data
+}
```

Comment:
Nice, you created the separate file to declare the object. and using this information in your code